doc/general_documentation/adding_a_command.md

```diff
@@ -51,6 +51,11 @@ The script will copy the generated files to the correct locations. Please make s
 
 Once everything compiles, make sure you commit and push your changes.
 
+
 # Add to Executive
 
 The last step is to add the command to the executive. Please make sure you have a full understanding of the executive and operating state classes before adding the command. If you have questions, please contact Katie Hamilton.
+
+\htmlinclude AstrobeeCommandDictionary.html
```

Comment:
This should be somewhere else. People are not going to look at the how to add a command documentation to find the command dictionary. I think we should put the command dictionary in its own subpage under the Astrobee Usage page.

Reply:
I put it here https://marinagmoreira.github.io/astrobee/html/command_dictionaty.html under "Astrobee useage"